### PR TITLE
chore: Delete useless files

### PR DESCRIPTION
**Pull request by gaocegege**
I am not sure why we have setup.py. But it seems that we can delete it safely.

/assign @YujiOshima 

Signed-off-by: Ce Gao <gaoce@caicloud.io>

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/493)

### CODE REVIEW

File: py/study_job_client.py [hunk 1]
```diff
@@ -1,160 +0,0 @@
-# Copyright 2019 Google LLC
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
```
Comment (hougangliu) [inline py/study_job_client.py]: this file is used in https://github.com/kubeflow/pipelines/blob/master/components/kubeflow/katib-launcher/Dockerfile

Comment (gaocegege) [reply py/study_job_client.py]: OK, then we need to keep it. I will add a comment about it.

**Comment by gaocegege** [comment]
@hougangliu PTAL again.

File: setup.py [hunk 2]
```diff
@@ -1,4 +0,0 @@
-from setuptools import setup
```
Comment (hougangliu) [inline setup.py]: this file used in python_tests.py, but not sure if it is useful now

Comment (gaocegege) [reply setup.py]: I am trying if it works after the deletion.

**Comment by gaocegege** [comment]
It seems that we cannot delete it...

**Pull request closed without merging**